core/rawdb/accessors_skipped_txs.go

```diff
@@ -157,6 +157,24 @@ func WriteSkippedTransaction(db ethdb.Database, tx *types.Transaction, traces *t
 	}
 }
 
+func ResetSkippedTransactionTracesByHash(db ethdb.Database, txHash common.Hash) {
+	mu.Lock()
+	defer mu.Unlock()
```

Comment:
No need to lock, the other method only locks because all invocations to `WriteSkippedTransaction` (with a capital W) share a single counter.